src/cdk/v2/destinations/bloomreach/procWorkflow.yaml

```diff
@@ -24,22 +24,42 @@ steps:
     template: |
       $.context.messageType = .message.type.toLowerCase();
 
+  - name: mappedToDestination
+    template: |
+      .message.context.mappedToDestination || false;
+
   - name: validateInput
```

Comment:
We can have rootsteps as validateInput
Then we can have 2 sub steps :- validateContactAndEventsInput, validateCatalogInput